packages/terser/test/sourcemap-arg.spec.js

```diff
@@ -0,0 +1,19 @@
+const {processSourceMapArg} = require('npm_bazel_terser/index')
+
+describe('processSourceMapArg', () => {
+    it('return a new array ref', () => {
+  const args = [];
+  expect(processSourceMapArg(args)).not.toBe(args)
+    })
+
+it('should strip the base and leading / from the output url', () => {
```

Comment:
why is the formatter not working for you? there should be a husky-installed pre-commit hook to check that you formatted your changes

Reply:
Im pretty sure thats what the formatter wanted to do.
I even ran it manually, as in `clang-format` instead of `git-clang-format`

Reply:
ah! it was formatting correctly, but I forgot some semi colons - im use to prettier isnerting them for me